website/docs/r/access_rule.html.markdown

```diff
@@ -10,6 +10,16 @@ description: |-
 
 This resource allows you to create and manage access rules, to give users and/or service accounts permissions on directories.
 
+## Example Usage
+
+```terraform
+resource "secrethub_access_rule" "example_app" {
+  account_name = "${secrethub_service_aws.demo_app.id}"
```

Comment:
This uses `example_app` and `demo_app`. These names don't have to be the same, but it might be nice to be consistent in naming.